collated/main/nahtanojmil.md

```diff
@@ -0,0 +1,361 @@
+# nahtanojmil
+###### \java\seedu\address\logic\commands\RemarkCommand.java
+``` java
+/**
+ * Changes the remark of an existing person in the address book.
+ */
+public class RemarkCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "remark";
+    public static final String COMMAND_ALIAS = "rm";
+
+    public static final String MESSAGE_ADD_REMARK_SUCCESS = "Added remark to person: %1$s";
+    public static final String MESSAGE_DELETE_REMARK_SUCCESS = "Removed remark from person: %1$s";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " : edits the remarks of a person "
+            + "by the index from the last shown list. Old remarks will be overwritten by new "
+            + "ones entered.\n"
+            + "parameters: INDEX (positive integer)" + PREFIX_REMARK + "[REMARK]\n"
+            + "Example: " + COMMAND_WORD + "3 " + PREFIX_REMARK + "likes to eat!";
+
+    private Index index;
+    private Remark stringRemark;
+
+
+    public RemarkCommand(Index index, Remark remark) {
+        if (index != null && remark != null) {
+            this.index = index;
+            this.stringRemark = remark;
+        }
+    }
+
+    @Override
+    protected CommandResult executeUndoableCommand() throws CommandException {
+        List<ReadOnlyPerson> lastShownList = model.getFilteredPersonList();
+
+        if (index.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        }
+
+        ReadOnlyPerson toBeEdited = lastShownList.get(index.getZeroBased());
+        Person editedPerson = new Person(toBeEdited.getName(), toBeEdited.getPhone(), toBeEdited.getParentPhone(),
+                toBeEdited.getEmail(), toBeEdited.getAddress(), toBeEdited.getFormClass(),
+                toBeEdited.getGrades(), toBeEdited.getPostalCode(), stringRemark, toBeEdited.getTags());
+
+        try {
+            model.updatePerson(toBeEdited, editedPerson);
+        } catch (DuplicatePersonException e) {
+            throw new CommandException(MESSAGE_DUPLICATE_PERSON);
+        } catch (PersonNotFoundException f) {
+            throw new AssertionError("Person must be from last filtered list");
+        }
+        return new CommandResult(generateSuccessMessage(editedPerson));
+    }
+
+    /**
+     * @param editedPerson takes in a edited person and
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)